runway/commands/runway/test.py

```diff
@@ -78,6 +78,8 @@ def execute(self):
                 # tool it is wrapping.
                 if not isinstance(err, SystemExit):
                     traceback.print_exc()
+                elif err.code == 0:
+                    next  # Tests calling sys.exit(0) don't indicate failure
```

Comment:
I think you are looking for `continue` here

```suggestion
                    continue
```